PixelHunter1995/FontManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework.Content;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace PixelHunter1995
+{
+    class FontManager
+    {
+        private Dictionary<string, SpriteFont> Fonts;
+        private static FontManager instance;
+
+        public static FontManager Instance
+        {
+            get
+            {
+                if (instance == null)
+                {
+                    instance = new FontManager();
+                }
+                return instance;
+            }
+        }
+
+        private FontManager() {}
```

Comment:
What does this line do?

Reply:
Makes the constructor private, which it should be with a singleton.